VALUES_AND_PRIORITIZAION.md

```diff
@@ -0,0 +1,47 @@
+# Values and Priorities
+
+> Document Status: Proposal, Work In Progress
+
+## Context
+
+Define the `Technical` values shared by the project and their relatively priority
+that are applied when making tradeoffs.
+
+
+## Values and priority level
+
+This is only a subset of the overall prioties, listing only those which are at the
+top of the priority list. Anything on this list is very important for the project
+despite the relative priorities shown.
+
+- Prio 1 - Developer experience
+- Prio 2 - Stability
+- Prio 3 - Operational qualities
+- Prio 4 - Technology and API Currency
```

Comment:
I would like to add maintainer experience to this list. IMO, without maintainers none of the above can happen.